x-pack/legacy/plugins/siem/public/components/ml/anomaly_score.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+import { EuiPopover, EuiDescriptionList, EuiFlexItem, EuiIcon } from '@elastic/eui';
+import styled from 'styled-components';
+import { NarrowDateRange, Anomaly } from './types';
+import { DraggableScore } from './draggable_score';
+import { escapeDataProviderId } from '../drag_and_drop/helpers';
+import { createDescriptionsList } from './create_descriptions_list';
+
+interface Args {
+  startDate: number;
+  endDate: number;
+  narrowDateRange: NarrowDateRange;
+  jobKey: string;
+  index?: number;
+  score: Anomaly;
+  interval: string;
+}
+
+const Icon = styled(EuiIcon)`
+  vertical-align: text-bottom;
+  cursor: pointer;
+`;
+
+export const AnomalyScore = React.memo<Args>(
+  ({ jobKey, startDate, endDate, index = 0, score, interval, narrowDateRange }): JSX.Element => {
+    const [isOpen, setIsOpen] = useState(false);
+    return (
+      <>
+        <EuiFlexItem grow={false}>
+          <DraggableScore
+            id={escapeDataProviderId(`anomaly-scores-${jobKey}`)}
+            index={index}
+            score={score}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiPopover
+            data-test-subj="anomaly-score-popover"
+            id="anomaly-score-popover"
+            isOpen={isOpen}
+            onClick={() => setIsOpen(!isOpen)}
```

Comment:
Neat! I've been putting this handler on the `button` prop. I like this better though :)